scripts/hallucinate_fragalysis.py

```diff
@@ -0,0 +1,589 @@
+"""
+Build library of ligands from a dataset of holo crystal structures docked to a
+different dataset of apo structures.
+"""
+import argparse
+from glob import glob
+import itertools as it
+import multiprocessing as mp
+from openeye import oechem, oedocking
+import os
+import pandas
+import pickle as pkl
+import re
+import sys
+
+sys.path.append(f"{os.path.dirname(os.path.abspath(__file__))}/../")
+from covid_moonshot_ml.datasets.utils import (
+    get_compound_id_xtal_dicts,
+    load_openeye_pdb,
+    load_openeye_sdf,
+    parse_fragalysis_data,
+    save_openeye_pdb,
+    save_openeye_sdf,
+    split_openeye_mol,
+)
+from covid_moonshot_ml.docking.docking import (
+    build_docking_system_direct,
+    run_docking,
+)
+from covid_moonshot_ml.modeling import du_to_complex, make_du_from_new_lig
+
+
+def check_output(d):
+    ## First check for result pickle file
+    try:
+        pkl.load(open(f"{d}/results.pkl", "rb"))
+    except Exception:
```

Comment:
You probably only want to catch if the file isn't found, right now this will also silently catch any errors when loading the pickle

Reply:
done here: https://github.com/choderalab/covid-moonshot-ml/commit/a76009b68bb85312676b27ca2fa3302ace0b25ae